spinner-lib/src/main/AndroidManifest.xml

```diff
@@ -2,7 +2,7 @@
     package="com.duchuyctlk.widget">
 
     <application
-        android:allowBackup="false"
+        android:allowBackup="true"
```

Comment:
If this attribute remains the same. Is the bug fixed? What is the role of `allowBackup` in solving `StackOverflowException` crash?